### DISCUSSION TITLE
webdav 无密码访问

### DISCUSSION THREAD

**Discussion by ptr7F** [AlistGo/alist#6376] [General]
用ffplay调试，有用户名和密码的情况下不能正常播放，alist的guest有webdav读取写入权限也要输入用户名密码。例如用guest，播放链接用如下进行连接：
http://guest:guest@localhost/dav
能不能弄个不需要输入用户和密码的模式？

**Comment by pongfcnkl** [comment]
把游客打开？

**Reply by ptr7F** [reply]
游客不就是指用户guest吗？webdav也要输入用户名和密码
